src/components/Footer.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import TwitterSVG from '../svg/Twitter';
+import FacebookSVG from '../svg/Facebook';
+import GithubSVG from '../svg/Github';
+import ExternalLinkSVG from '../svg/ExternalLink';
+import '../css/components/Footer.css';
+
+class Footer extends Component {
+
+  render() {
+    return (
+      <footer className="coa-Footer">
+        <div className="coa-section row">
+          <div className="col-xs-12">
+            <h3>AUSTIN.GOV</h3>
+          </div>
+          <div className="col-xs-12 col-md-6 last-md">
+            <p className="coa-Footer__body-text">
+              Austin.gov is a new website and a work in progress. For the full City of Austin website, visit <a href="http://austintexas.gov" target="_blank" aria-label="Opens in new window">austintexas.gov <ExternalLinkSVG size="13"/></a>. Learn more about the new website at <a href="http://projects.austintexas.io/projects/austin-digital-services-discovery/about/what-we-are-doing/" target="_blank" aria-label="Opens in new window">projects.austintexas.io <ExternalLinkSVG size="13"/></a>.
```

Comment:
when we use `target="_blank"` on an anchor tag, we should add `aria-label="Opens in new window"`. Wonder if there is a programmatic way to do that, or at least lint for it.
https://accessibility.18f.gov/

Reply:
sure thing!